move.go

```diff
@@ -59,6 +59,20 @@ type Move struct {
 	Priority  int
 	Power     int
 	Accuracy  int
+	metadata  MoveMeta
+}
+
+type MoveMeta struct {
+	MinHits       int
+	MaxHits       int
+	MinTurns      int
+	MaxTurns      int
+	Drain         int
+	Healing       int
+	CritRate      int
+	AilmentChance int
+	FlinchChance  int
+	StatChance    int
```

Comment:
These should probably just go in the `Move` struct (or `MoveData` in PR #256 )

Reply:
These should go in `MoveData` directly, and the `MoveMeta` type should not be necessary.